v03/index.js

```diff
@@ -23,7 +23,6 @@ module.exports = {
   BinaryHTTPEmitter,
   BinaryHTTPReceiver,
   HTTPUnmarshaller,
-  Cloudevent,
```

Comment:
This is a breaking change to the API. I think we need to branch a `v1.x` line before doing this, or simply leave this line in for backwards compat.

Reply:
Yes, this is a breaking change. I can change the commit message accordingly.
This would be a `2.0`, we don't need separate git branches for breaking changes, just major version changes.

Reply:
@grant I'm not sure you understand my proposal. I am not suggesting that we have branches for breaking changes.

Reply:
OK, let's talk about the breaking change issue separately and reconvene here.